lib/index.js

```diff
@@ -161,7 +162,7 @@ Quantcast.prototype.completedOrder = function(track) {
     event: 'refresh',
     labels: labels,
     revenue: String(revenue),
-    orderid: track.orderId(),
+    orderid: String(orderId),
```

Comment:
No need to move this into a variable. Unless you're doing it because the value will be used in multiple places, or because the alternative is confusing, avoid creating extra variables.